app/views/compte_mailer/alerte_admin.html.erb

```diff
@@ -3,5 +3,5 @@
          prenom: @compte.prenom,
          nom: @compte.nom,
          structure: @compte.structure.nom,
-         url_validation: "#{root_url}admin/comptes?q\\[statut_validation_eq\\]=en_attente"
+         url_validation: "#{root_url}admin/comptes/#{@compte.id}/edit"
```

Comment:
Je sais que ce n'est pas directement lié à cette PR mais ce n'est pas une bonne chose d'écrire en dur les URL de l'application, si on peut donc changer ce code par : 
```suggestion
         url_validation: edit_admin_compte_url(@compte)
```

Reply:
Lorsque je change l'url et que je lance les tests j'ai cette erreur : 
```Minitest::UnexpectedError:
       ActionView::Template::Error: undefined method `edit_admin_compte_path' for #<ActionView::Base:0x0000000006af18>
       Did you mean?  edit_admin_compte_path
``` 
Et quand je mets edit_admin_compte_path, j'ai la même erreur avec le `Did you mean? edit_admin_compte_url`

Reply:
Ok, on verra pour ce problème dans une autre PR